packages/itwin/property-grid/src/test/components/MultiElementPropertyGrid.test.tsx

```diff
@@ -9,7 +9,7 @@ import { PropertyRecord, PropertyValueFormat } from "@itwin/appui-abstract";
 import { KeySet } from "@itwin/presentation-common";
 import { PresentationLabelsProvider, PresentationPropertyDataProvider } from "@itwin/presentation-components";
 import userEvents from "@testing-library/user-event";
-import { AncestorsNavigationControls, MultiElementPropertyGrid, PropertyGridManager } from "../../property-grid-react";
+import { AncestorsNavigationControls, MultiElementPropertyGrid, PropertyGridManager, TelemetryContextProvider } from "../../property-grid-react";
```

Comment:
Can you fix this to not import from barrel file?

Reply:
Fixed.